### PR TITLE
docs and release changelog notes 

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: docs/data-sources/opensearch.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Opensearch Data Source
+
+The Opensearch data source provides information about the existing Aiven Opensearch service.
+
+## Example Usage
+
+```hcl
+data "aiven_opensearch" "es1" {
```
Comment (mhoffm-aiven) [inline docs/data-sources/opensearch.md:8]: should probably be "os1"

File: docs/resources/opensearch_acl_config.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Opensearch ACL Config Resource
+
+The Opensearch ACL Config resource allows the configuration of ACL management on an Aiven Opensearch service.
+
+## Example Usage
+
+```hcl
+resource "aiven_opensearch_acl_config" "es-acl-config" {
```
Comment (mhoffm-aiven) [inline docs/resources/opensearch_acl_config.md:8]: here too

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]
lgtm otherwise though

File: docs/resources/opensearch_acl_rule.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Elasticsearch ACL Rule Resource
```
Comment (mhoffm-aiven) [inline docs/resources/opensearch_acl_rule.md:1]: here too

**Review by mhoffm-aiven — DISMISSED** [review]
found one more

File: docs/resources/opensearch_acl_rule.md (hunk 3 above)
Comment (ivan-savciuc) [reply docs/resources/opensearch_acl_rule.md]: fixes

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**